src/value/from.rs

```diff
@@ -512,6 +512,45 @@ impl From<Object> for Value {
     }
 }
 
+impl From<serde_json::Value> for Value {
+    #[inline]
+    fn from(val: serde_json::Value) -> Self {
+        match val {
+            serde_json::Value::Null => Value::new_null(),
+            serde_json::Value::Bool(b) => Value::new_bool(b),
+            serde_json::Value::Number(n) => {
+                if let Some(u) = n.as_u64() {
```

Comment:
should care about `float_roundtrip` feature. 

if we enable it, the serde_json should also enable  its`float_roundtrip` feature, and sonic_rs will make numbers as `RawNumber`.